src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/Networking/UvWriteReq.cs

```diff
@@ -31,6 +31,13 @@ public UvWriteReq(ILibuvTrace logger) : base(logger)
         {
         }
 
+        public override void Init(LibuvThread thread)
+        {
+            Init(thread.Loop);
+
+            base.Init(thread);
+        }
+
         public void Init(UvLoopHandle loop)
```

Comment:
We should make the `Init(UvLoopHandle loop)` method private, so it doesn't accidentally get called causing a Req object to not be tracked. That would be a really easy mistake to make.